packages/orchestration/src/util/translation.ts

```diff
@@ -0,0 +1,30 @@
+import type { SAPDocumentTranslation } from '../client/api/schema/index.js';
+import type { TranslationConfigParams } from '../orchestration-types.js';
+
+/**
+ * Convenience function to build a document translation configuration for orchestration service.
+ * @param config - Config for SAP Document Translation service.
+ * The target language is mandatory, while source language will be auto-detected if not provided.
+ * See https://help.sap.com/docs/translation-hub/sap-translation-hub/supported-languages-6854bbb1bd824ffebc3a097a7c0fd45d?state=DRAFT for list of supported languages.
+ * @returns A `SAPDocumentTranslation` configuration object.
+ * @example "buildOrchestrationTranslationConfig({source_language: 'de-DE', target_language: 'en-US'})"
+ */
+export function buildOrchestrationTranslationConfig(
+  config?: TranslationConfigParams
+): SAPDocumentTranslation {
+  if (!config || !config.target_language || !config.target_language.trim()) {
```

Comment:
[req] What about checking `!config?.target_language`?

[pp] I don't have strong opinion for whether we should add some sanity check by using `trim()` or maybe some other utilities. Because orchestration currently also doesn't do this and just forward the requests to the translation service. So technically empty string is just the same as any invalid string and orchestration service should do the check and return us something meaningful (other than the current 500). Already talked to orchestration and they will figure it out.